README.md

```diff
@@ -24,13 +24,13 @@ Pending publication on the PyPI repository, a compiled package can be installed
 this URL:
 
 ```bash
-pip install https://s3-us-west-2.amazonaws.com/databricks-tjhunter/koala/databricks_koala-0.0.5-py3-none-any.whl
+pip install https://s3-us-west-2.amazonaws.com/databricks-tjhunter/koalas/databricks_koalas-0.0.5-py3-none-any.whl
```

Comment:
TODO: need to correct the url.
@thunterdb what will the correct url here be like?

Reply:
Maybe keep the old URL for now, and commit this. We can update the README once we change the URL.